### PR TITLE
meta: ingore the bin dir

### PR DESCRIPTION
**Pull request by vincenzopalazzo**
I just noted this while working on https://github.com/ACINQ/phoenixd/pull/12

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,7 @@
 .gradle
 build/
 .cxx
+bin
```
Comment (pm47) [inline .gitignore:4]: How are you producing a `bin` directory?

Comment (vincenzopalazzo) [reply .gitignore:4]: I ran `./gradlew linuxX64DistZip`

Comment (pm47) [reply .gitignore:4]: That's strange, there is no reason why a `bin` directory would be created by this command. All build outputs should be under `build/`.

The only explanation I can think of is unzipping a jvm build, which does contain `bin` and `lib` directories.

Comment (vincenzopalazzo) [reply .gitignore:4]: Ok I look now at the code and is strange yes, I will try to see if I can reproduce in a fresh env this `bin` stuff.

**Comment by vincenzopalazzo** [comment]
Closing this till I am not able to reproduce it in a fresh env

**Pull request closed without merging**